### PR TITLE
kata automation support for 4.8 release

### PR DESCRIPTION
**Pull request by pruan-rht**
@liangxia @cpmeadors PTAL

### CODE REVIEW

File: lib/openshift/kata_config.rb [hunk 1]
```diff
@@ -6,7 +6,12 @@ class KataConfig < ClusterResource
     RESOURCE = 'kataconfigs'
 
     def install_completed_node_count(user: nil, cached: true, quiet: false)
-      raw_resource(user: user, cached: cached, quiet: quiet).dig('status', 'installationStatus', 'completed', 'completedNodesCount')
+      raw_resource(user: user, cached: cached, quiet: quiet).dig('status', 'installationStatus', 'completed', 'completedNodesList')
```
Comment (akostadinov) [inline lib/openshift/kata_config.rb:9]: Maybe needed to use `#count` here.

```
      raw_resource(user: user, cached: cached, quiet: quiet).dig('status', 'installationStatus', 'completed', 'completedNodesList')&.count || 0
```

Comment (akostadinov) [reply lib/openshift/kata_config.rb:9]: Otherwise seems good.

Comment (pruan-rht) [reply lib/openshift/kata_config.rb:9]: @akostadinov, good suggestion, while doing this PR, it actually discovered a [regression](https://issues.redhat.com/browse/KATA-730).  So this method will go back to `completeNodesCount` eventually.

**Comment by pruan-rht** [comment]
@akostadinov ok, updated.  Thx again!

**Pull request merged**